### DISCUSSION TITLE
最新的开发版无法使用讯飞插件

### DISCUSSION THREAD

**Discussion by 342620** [jing332/tts-server-android#55] [General]
在最新的开发版【0.6_202303282204】中，使用讯飞插件会报错：org.mozilla.javascript.WrappedException: Wrapped java.lang.Exception: Body is not a Bytes, HTTP-200=OK (<Unknown source>#36) in <Unknown source> at line number 36

**Comment by 342620** [comment]
此问题已经得到优化，谢谢您

**Reply by jing332** [reply]
有问题上Issue提，这里不会第一时间邮件通知。
